tagstudio/src/qt/modals/tag_database.py

```diff
@@ -18,17 +19,21 @@
 from src.qt.widgets.tag import TagWidget
 from src.qt.modals.build_tag import BuildTagPanel
 
+# Only import for type checking/autocompletion, will not be imported at runtime.
+if typing.TYPE_CHECKING:
+    from src.core.library import Tag
+    from src.qt.ts_qt import QtDriver
+
 
 class TagDatabasePanel(PanelWidget):
     tag_chosen = Signal(int)
 
-    def __init__(self, library):
+    def __init__(self, library: "Library", driver: "QtDriver"):
```

Comment:
Literal string `"Library"` and `"QtDriver"` as the type looks like a typo.

Reply:
The string literals are there to prevent NameErrors, since QtDriver is not actually imported at runtime (nor should Library be, just caught that) and is only under `if typing.TYPE_CHECKING:` for type checking. It's quirky, but it's what [PEP 484](https://peps.python.org/pep-0484/#forward-references) suggests